dev/com.ibm.websphere.appserver.features/visibility/protected/com.ibm.websphere.appserver.transaction-1.2.feature

```diff
@@ -24,7 +24,7 @@ IBM-API-Service: com.ibm.wsspi.uow.UOWManager, \
  com.ibm.websphere.appserver.containerServices-1.0, \
  com.ibm.websphere.appserver.javax.annotation-1.2; ibm.tolerates:=1.3; apiJar=false, \
  com.ibm.websphere.appserver.anno-1.0, \
- com.ibm.websphere.appserver.eeCompatible-8.0; ibm.tolerates:="6.0,7.0"
+ com.ibm.websphere.appserver.eeCompatible-7.0; ibm.tolerates:="6.0,8.0"
```

Comment:
I think it needs to be 6.0 tolerates 7.0, 8.0 actually like transaction 1.1.  This will be similar to other dependencies like servlet where we depend on 3.0 and tolerate 3.1 and 4.0.